bin/import_gram_vaani.py

```diff
@@ -0,0 +1,302 @@
+import os
+import csv
+import sys
+import math
+import urllib
+import logging
+import argparse
+import subprocess
+from os import path
+from pathlib import Path
+
+import swifter
+import pandas as pd
+from sox import Transformer
+
+from util.text import validate_label
+
+
+__version__ = "0.1.0"
+_logger = logging.getLogger(__name__)
+
+
+MAX_SECS = 10
+BITDEPTH = 16
+N_CHANNELS = 1
+SAMPLE_RATE = 16000
+
+DEV_PERCENTAGE = 0.10
+TRAIN_PERCENTAGE = 0.80
+
+
+def parse_args(args):
+    """Parse command line parameters
+    Args:
+      args ([str]): Command line parameters as list of strings
+    Returns:
+      :obj:`argparse.Namespace`: command line parameters namespace
+    """
+    parser = argparse.ArgumentParser(
+        description="Imports GramVaani data for Deep Speech"
+    )
+    parser.add_argument(
+        "--version",
+        action="version",
+        version="GramVaaniImporter {ver}".format(ver=__version__),
+    )
+    parser.add_argument(
+        "-v",
+        "--verbose",
+        action="store_const",
+        required=False,
+        help="set loglevel to INFO",
+        dest="loglevel",
+        const=logging.INFO,
+    )
+    parser.add_argument(
+        "-vv",
+        "--very-verbose",
+        action="store_const",
+        required=False,
+        help="set loglevel to DEBUG",
+        dest="loglevel",
+        const=logging.DEBUG,
+    )
+    parser.add_argument(
+        "-f",
+        "--file",
```

Comment:
"file" seems very generic. How about something more descriptive like "source_csv"?
I'd prefer a positional argument, as it feels more natural for a required setting.

Reply:
OK